### PR TITLE
Milestone 1: Mobile Version Portfolio

### PR DESCRIPTION
**Pull request by Dinma-Faith**
##Mobile View of Portfolio:

In this milestone I; 
-Created HTML and CSS file
-Created Img file and added images in it 
-Created a linters file to detect code errors

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.wrap {
+  background-image: url(./img/headerbg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  max-width: 400px;
+  height: 90vh;
+  margin: 5vh auto 5vh auto;
+}
```
Comment (Amrhub) [inline style.css:14]: - [ ] Let's make sure our background image is on the `headline` section i.e `header` excluded to match the design, if you notice in Figma design back ground image starts after the navbar/header tag

Your Implementation | Figma design 
:-----:|:-----:
image | image

Comment (Amrhub) [inline style.css:14]: - [ ] Let's make sure we don't have margins to the top to match design the space in the design to the top is (I know can be confusing) just there to memic as this website was opened via mobile

Your implementation | Figma Design
:------:|:-----:
image | image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  display: inline-flex;
+  align-items: center;
+  color: #172b4d;
+  opacity: 1.5;
+}
+
+.connect a {
+  text-decoration: none;
+  opacity: 0.5;
+  margin-left: 1rem;
```
Comment (Amrhub) [inline style.css:56]: - [ ] In order to match Figma design we need to make sure that our `let's connect doesn't have any margin left so it starts with the rest of the content on this page.
I made a red straight line to show you how it should be aligned in comparison with Figma design

Your implementation | Figma Design
:------:|:-----:
image | image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+.social-links ul {
+  display: flex;
+  align-items: flex-start;
+  width: 100%;
+  list-style: none;
+}
+
+.media-img li {
+  padding: 0.5rem;
+  margin: 0.5rem;
```
Comment (Amrhub) [inline style.css:68]: - [ ] As previously mentioned above, Let's make sure our social media links starts with the start of the rest of content

Your implementation | Figma Design
:------:|:-----:
image | image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @Dinma-Faith,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Professional documentation ✔️ 
Clean code ✔️ 

## Required Changes ♻️
_Check the comments under the review._

Most of the changes required have to do with following the Figma design, other than that you have made a great effort and fantastic work




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ReshyResh — APPROVED** [review]
# Hi @Dinma-Faith , 

### Good job addressing the changes requested by the previous reviewer!
Your project is complete fow now, but keep in mind that, as the other reviewer said, you should avoid hard-coded values for the width of the website, as later on you will be tasked with making it responsive for desktop screens as well!  

#### There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**